### DISCUSSION TITLE
Dev taking time off?

### DISCUSSION THREAD

**Discussion by Ricaz** [aeharding/voyager#724] [General]
Like 3 weeks ago I got several updates _per day_ in the app. What happened?

Not trying to be a choosing beggar, just wondering if dev is taking some time off after some HEAVY open source contributing :)

**Comment by JonoHall** [comment]
Last release had the comment:

> Gone fishing! 🐟 Development will slow for the next 1.5 weeks.

Guess they extended the fishing trip.

**Comment by sharunkumar** [comment]
How I feel right now

![image](https://github.com/aeharding/voyager/assets/715417/c72f8580-4970-4449-8e9f-c0d54776ff70)
